Firebase/Core/FIRApp.m

```diff
@@ -549,6 +557,10 @@ + (NSString *)firebaseUserAgent {
   }
 }
 
++ (BOOL)hasSwiftRuntime {
+  return objc_lookUpClass("Swift._SwiftObject") != nil;
```

Comment:
Great! Is there a resource you could add (even as a PR description) describing where you found this information and why this will work?

Reply:
Good point! I added comments to the code. The idea originated from the [internal discussion](https://groups.google.com/a/google.com/g/appledev-discuss/c/QTGnFJPJzOQ/m/N74DXXcpCwAJ).

Reply:
Will this always be true when running on ios 12.4 and newer, even if there is no swift?

Reply:
It was not on simulator. I think even if it is, it will be fine. I assume, the main question we would like to answer is "how many applications/devices will have size bump if Swift is added". 

But anyways, let me check on a real device just in case.

Reply:
For me the class `Swift._SwiftObject` is found only when a swift file present in the project on my iPhone iOS 13.2.3

Reply:
Interesting. So we'll need to cross-reference the Swift flag with the device iOS version to know where Swift causes a size impact.

Reply:
Yeah, looks like it.